gnocchi-cli/src/main/scala/net/fnothaft/gnocchi/cli/RegressPhenotypes.scala

```diff
@@ -107,6 +112,11 @@ class RegressPhenotypes(protected val args: RegressPhenotypesArgs) extends BDGSp
     val a = args.oneTwo
     //    println(s"\n\n\n\n\n\n oneTwo: $a \n\n\n\n\n\n\n\n")
 
+    import Timers._
+    Metrics.initialize(sc)
```

Comment:
this is also handled in bdg-utils for the `-print_metrics` argument

https://github.com/bigdatagenomics/utils/blob/master/utils-cli/src/main/scala/org/bdgenomics/utils/cli/BDGCommand.scala#L68